smac/optimizer/ei_optimization.py

```diff
@@ -259,13 +243,15 @@ def _one_iter(
             start_point: Configuration,
             *args
     ) -> Tuple[float, Configuration]:
+        N_STEPS_PLATEAU_WALK = 10
```

Comment:
This should not be defined in here, but somewhere else, e.g. SMAC configuration.